tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -495,7 +495,7 @@ def configureConnections():
         scrapeTrxGenTrxSentDataLogs(trxSent, self.trxGenLogDirPath, self.ptbConfig.quiet)
         if len(trxSent) != self.ptbConfig.expectedTransactionsSent:
             print(f"ERROR: Transactions generated: {len(trxSent)} does not match the expected number of transactions: {self.ptbConfig.expectedTransactionsSent}")
-        blocksToWait = 2 * self.ptbConfig.testTrxGenDurationSec + 10
+        blocksToWait = self.data.startBlock + (2 * self.ptbConfig.testTrxGenDurationSec) + 10
```

Comment:
`blocksToWait` is meant to be the number of blocks to wait after a given block. so the `startBlock` was not included on purpose.  You should see the `startBlock` being added into the call to `waitForTrasactionsInBlockRange` in `performance_test_basic.py` and then in `Node.py` `waitForTransactionsInBlockRange` -> `overallEndBlock = startBlock + maxFutureBlocks`

Reply:
Ok. I looked at that wrong. It must have been failing because it took  more than 10 blocks for the trx generators to fire up as it was consistently failing on not getting the trxs from the blocks when they were there. Since the function is called `waitForTransactionsInBlockRange, I think I'll change it to take a range instead of start and offset.